consent_manager/index.rst

```diff
@@ -2,6 +2,4 @@ Consent Manager
 ================
 
 .. toctree::
-  :maxdepth: 2
-
-  api
+  js_api/index
```

Comment:
Change of page URL may break links to it. Are you sure you don't need any redirect / message what happened with that page?

Reply:
good point, added api.rst back with redirect